rakelib/release.rb

```diff
@@ -3,6 +3,11 @@ def git_directory
   File.directory?(git_dir) && git_dir
 end
 
+def in_git?
+  git_dir = File.expand_path "../../.git", __FILE__
+  File.exist?(git_dir) && git_dir
```

Comment:
Why is there the extra `&& git_dir` here? As far as I'm aware `File.expand_path` never returns nil.